SU2_PY/shape_optimization.py

```diff
@@ -49,7 +49,7 @@ def main():
                       help="try to restart from project file NAME", metavar="NAME")
     parser.add_option("-n", "--partitions", dest="partitions", default=1,
                       help="number of PARTITIONS", metavar="PARTITIONS")
-    parser.add_option("-g", "--gradient", dest="gradient", default="CONTINUOUS_ADJOINT",
+    parser.add_option("-g", "--gradient", dest="gradient", default="DISCRETE_ADJOINT",
```

Comment:
I think changing default values is a big step. I would actually recommend to not change it for now.

Reply:
I agree - unfortunately most users will not have compiled with AD and may find this confusing.

Reply:
Based on my experience DISCRETE_ADJOINT is the only one that really works with turbulent flows... and that is what most of the users are expecting. No reason for not compiling the software with AD... it should be the default.

Reply:
I am the last person who has something against this kind of change :) Still, in order for that to be default I should perhaps update the preconfigure script to work more reliably (especially when SU2 is not cloned using github ...)

Reply:
Discrete adjoint is going to be the default for the tutorials. Continuous adjoint is working great for inviscid problems but... from the user point of view you are looking for something that works in any circumstance. The community should start using AD by default and we need to push that. In the tutorial we will explicitly indicate DISCRETE or CONTINUOUS